src/media/css/games/listing.styl

```diff
@@ -0,0 +1,99 @@
+@import '../lib';
+
+
+.web-games-are-awesome {
+    align-items: center;
+    background: $action-success-tapped;
+    color: $greyscale-white;
+    display: flex;
+    flex-direction: column;
+    font-weight: 400;
+    font-size: 20px;
+    justify-content: center;
+    padding: 50px 0 40px;
+    width: 100%;
+
+    p,
+    a {
+        text-align: center;
+        width: 50%;
+    }
+    p {
+        margin-bottom: 20px;
+    }
+    a {
+        color: $greyscale-white;
+        text-decoration: underline;
+    }
+}
+
+
+.games-listing-container {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 50px;
+
+    +media--base-desktop() {
+        flex-direction: row;
+        justify-content: space-between;
+    }
+}
+
+.games-listing-nav {
+    align-items: center;
+    background: $greyscale-white;
+    display: flex;
+    flex-direction: column;
+    height: 300px;
+    padding: 30px 30px 40px 30px;
+    width: 100%;
+
+    +media--base-desktop() {
+        align-items: flex-start;
+        height: 320px;
+        margin: 5px 5px 0 0;
+        width: 240px;
+    }
+    li {
+        display: inline-block;
+        list-style: none;
+        font-weight: 600;
+        transition: background .1s;
+        width: fit-content;
+
+        a {
+            background: none;
+            border-radius: 10px;
+            color: $greyscale-black;
+            display: inline-block;
+            font-size: 20px;
+            margin: 6px 0;
+            padding: 5px 10px;
+            transition: color .1s;
+        }
+        &:hover,
+        &[data-active="true"] {
```

Comment:
This is nice. I tried coming up with a better way to do this, but couldn't. Well done.